charts/backstage/values.yaml

```diff
@@ -1,4 +1,8 @@
 global:
+  dynamic:
+    includes: ['dynamic-plugins.default.yaml']
```

Comment:
Please comment on where is this file located (basepath for `includes`) and that it only works with backstage-showcase image.

Reply:
Done